### DISCUSSION TITLE
Continuing the conversation after `cmd+shift+i`

### DISCUSSION THREAD

**Discussion by valdemartti** [JudiniLabs/code-gpt-docs#66] [Ideas]
Hey! I would like to see (if it doesn't already exist), a functionality to continue the conversation with the AI about it's suggestion for a comment I wrote, after I've pressed `cmd+shift+i`. I tried to continue the conversation in the chat tab, but it didn't remember the context I was in.

Is there something that can be done with this, maybe it could use the same context for both the chat and the `cmd+shift+i`? Also, the `cmd+shift+i` response did not return a Markdown tab or a preview, it returned the same type of file that was currently chosen, making ESLint go crazy on it.

Thanks!

**Comment by davila7** [comment]
This is good!

I will add it to the next update. So that the chat is the main tool with which you interact.
Thanks for the suggestion

**Comment by davila7** [comment]
Moved to discussion
